README.md

```diff
@@ -128,6 +128,27 @@ pnpm export adaptor-docs
 
 This will build adaptor-docs into a tarball and install it directly into Lightning (assuming that `Lightning` is a sibling dir of `kit`).
 
+## Code Formatting with Prettier
```

Comment:
The documentation here (between lines 131 and 137) says the same thing twice, and the editor integration section doesn't really add much.

Documentation should be clear and concise. Please have a go at simplifying this!